lib/isorender/dom-shims.js

```diff
@@ -27,28 +27,54 @@ import requestAnimationFrame from 'raf';
 // is already there
 global.requestAnimationFrame = global.requestAnimationFrame || requestAnimationFrame;
 
+function walkDomTree(root, callback) {
```

Comment:
let's get an explanatory comment in here e.g.
```suggestion
// run a callback for every node of the DOM (sub)tree rooted
// at the given root node
function walkDomTree(root, callback) {
```